src/grammar/cue.bnf

```diff
@@ -204,12 +206,12 @@ Arguments      ::= "(" [ ( Argument { comma Argument }* ) [ comma ] ] ")" {exten
 // fixme Slice is missing
 
 // https://cuelang.org/docs/references/spec/#operators
-Expression ::= UnaryExpr | BinaryExpr // fixme extra root?
+Expression ::= UnaryExpr | UnificationBinaryExpr | BinaryExpr // fixme extra root?
```

Comment:
Was `UnificationBinaryExpr` only introduced to be able to use it in structure view?
You could also expose left and right tokens of `BinaryExpr`.
Alternatively you could keep `UnificationBinaryExpr`, but make it `extend = BinaryExpr` and expose left and right tokens in `BinaryExpr` instead. This way `UnificationBinaryExpr` is just a subtype of `BinaryExpr`.